metriq_gym/benchmarks/bseq.py

```diff
@@ -192,3 +197,34 @@ def poll_handler(
             largest_connected_size=lcs,
             fraction_connected=lcs / job_data.coloring.num_nodes,
         )
+
+    def upload_handler(
+        self, job_data: BenchmarkData, result_data: BenchmarkResult, dispatch_time: datetime, submission_id: int, platform_id: int,
+    ) -> None:
+        client = MetriqClient(os.environ.get("METRIQ_CLIENT_API_KEY"))
+        task_id = 236 #BSEQ task ID
+        method_id = 426 #BSEQ method ID
+        client.submission_add_task(submission_id, task_id)
+        client.submission_add_method(submission_id, method_id)
+        client.submission_add_platform(submission_id, platform_id)
+        result_create_request = ResultCreateRequest(
+            task = str(task_id),
+            method = str(method_id),
+            platform = str(platform_id),
+            isHigherBetter = str(True),
+            metricName = "",
+            metricValue = str(0),
+            evaluatedAt = dispatch_time.strftime("%Y-%m-%d"),
+            qubitCount = str(job_data.num_qubits),                                         # type: ignore[attr-defined]
```

Comment:
@WrathfulSpatula Right now the model of metriq.info for what is essentially the list of _parameters_ of a benchmark is quite static, and it's defined in https://github.com/unitaryfund/metriq-api/blob/99be3d229df21f4d186cc25ae04e610229833676/metriq-api/models/resultModel.js#L34
We want this to be more flexible to allow more parameters. For example, in some benchmarks we have `trial`, `num_circuits`, `type_of_twirling`, etc...
As far as I can see, we have a degree of flexibility in the fact that we can dump all the parameters in the Notes field. However, we may want something slightly more structured like a free JSON field, instead of a TEXT.